src/core/Entry.cpp

```diff
@@ -325,25 +325,28 @@ QString Entry::totp() const{
 
         int rc = oath_init();
```

Comment:
There is virtually no difference. If you really want to be sure that they don't change it to something that doesn't work with rvalues (highly unlikely), you can always make an lvalue from it before using Q_UNUSED. The point is just that we don't want compiler warnings about ignored return values and Q_UNUSED can get rid of them. Whether you apply it directly to the function call or create an lvalue first, I don't care.